rust/repo/examples/default/output/repository/Cargo.toml

```diff
@@ -23,11 +23,11 @@ dev-version-ext = "beta.0"
 consolidate-commits = true
 consolidate-pushes = true
 tag-name = "v{{version}}"
-
-# The file names in this key are relative to the each crate that gets released.
-# So we only need one `README.md` entry if all the README's follow a
-# common format and are always next to the Cargo.toml for their respective
-# package.
 pre-release-replacements = [
-    {file="README.md", search="mc-[a-z-]+/[0-9.]+", replace="{{crate_name}}/{{version}}"},
+    { file = "README.md", search = "mc-[a-z-]+/[0-9.]+", replace = "{{crate_name}}/{{version}}" },
+    { file = "CHANGELOG.md", search = "Unreleased", replace = "{{version}}", min = 1 },
+    { file = "CHANGELOG.md", search = "\\.\\.\\.HEAD", replace="...{{tag_name}}", exactly = 1 },
+    { file = "CHANGELOG.md", search = "ReleaseDate", replace= "{{date}}", min = 1},
+    { file = "CHANGELOG.md", search = "<!-- next-header -->", replace="<!-- next-header -->\n## [Unreleased] - ReleaseDate\n", exactly = 1 },
+    { file = "CHANGELOG.md", search = "<!-- next-url -->", replace="<!-- next-url -->\n[Unreleased]: https://github.com/mobilecoinfoundation/repository/compare/{{tag_name}}...HEAD", exactly = 1},
```

Comment:
:x: Sorry, I failed in my original comment message.

We don't want this in the root for workspaces, this will get executed for *each* crate, which means that it will look for a changelog in each crate's directory and fail, see https://github.com/mobilecoinfoundation/sgx/blob/main/core/build/Cargo.toml#L18

Thinking this should get placed in the "seed" crate with a similar comment.